DigitalLearningSolutions.Web/Services/UserService.cs

```diff
@@ -275,7 +275,7 @@ public void UpdateFailedLoginCount(UserAccount userAccount)
         public IEnumerable<DelegateUserCard> GetDelegateUserCardsForWelcomeEmail(int centreId)
         {
             return userDataService.GetDelegateUserCardsByCentreId(centreId).Where(
-                user => user.Approved && !user.SelfReg && string.IsNullOrEmpty(user.Password) &&
+                user => user.Approved && !user.SelfReg && (string.IsNullOrEmpty(user.Password) || !string.IsNullOrEmpty(user.Password)) &&

```

Comment:
I think this condition is not at all required. The condition (string.IsNullOrEmpty(user.Password) || !string.IsNullOrEmpty(user.Password)) will always return true because a Password can only be either empty or not empty. This means the condition will pass the filter regardless of the Password, effectively ignoring the Password condition entirely.

Reply:
Done